spanner/sessionclient.go

```diff
@@ -90,17 +89,18 @@ type sessionClient struct {
 	closed               bool
 	disableRouteToLeader bool
 
-	connPool      gtransport.ConnPool
-	database      string
-	id            string
-	userAgent     string
-	sessionLabels map[string]string
-	databaseRole  string
-	md            metadata.MD
-	batchTimeout  time.Duration
-	logger        *log.Logger
-	callOptions   *vkit.CallOptions
-	otConfig      *openTelemetryConfig
+	connPool             gtransport.ConnPool
+	database             string
+	id                   string
+	userAgent            string
+	sessionLabels        map[string]string
+	databaseRole         string
+	md                   metadata.MD
+	batchTimeout         time.Duration
+	logger               *log.Logger
+	callOptions          *vkit.CallOptions
+	otConfig             *openTelemetryConfig
```

Comment:
should we use `otel`?

Reply:
Sorry are you suggesting renaming the variable here, or want to ask if we should otel, 
If it's latter then yes we now want to use otel going forwards since opencensus will deprecate soon.

Reply:
I am suggesting to rename `otConfig` to `otelConfig`.